src/dso_api/dynamic_api/views/mvt.py

```diff
@@ -114,6 +114,7 @@ def setup(self, request, *args, **kwargs):
     def get(self, request, *args, **kwargs):
         kwargs.pop("dataset_name")
         kwargs.pop("table_name")
+        self.z = kwargs["z"]
```

Comment:
No risk for KeyError here?

Reply:
The URL format is fixed in this line:

https://github.com/Amsterdam/dso-api/blob/aaa702c82d3acb575d4629a0f4a00c1f10455a23/src/dso_api/dynamic_api/routers.py#L390